.eslintrc.js

```diff
@@ -83,7 +84,7 @@ module.exports = {
     // --- React
     "react/prop-types": "off",
     // we use 'logger' inside of hooks, gets flagged 
-    "react-hooks/exhaustive-deps": "off",
+    "react-hooks/exhaustive-deps": 1,
```

Comment:
does this generate compiler warnings and issues? if so, might be better to split off in a separate PR that also fixes the issues

Reply:
Good catch! This file is not part of this PR.